pytest_tests/helpers/k6.py

```diff
@@ -68,7 +67,7 @@ def process_dir(self) -> str:
     def k6_dir(self) -> str:
         if not self._k6_dir:
             self._k6_dir = self.shell.exec(
-                r"sudo find . -name 'k6' -exec dirname {} \; -quit"
+                r"sudo find / -name 'k6' -exec dirname {} \; -quit"
```

Comment:
But what's the problem here, really? Why do we have to `find` something?

Reply:
Here the code is looking for a directory with cloned https://github.com/nspcc-dev/xk6-neofs

Reply:
The test relies on some previous load node configuration where this repo was cloned under a home directory of an ssh user

Reply:
Let's fix this, there is no real reason to search for this directory, it should be known.

Reply:
Fixed